Sources/ApiVideoPlayer/ApiVideoPlayerController.swift

```diff
@@ -15,6 +15,7 @@ public class ApiVideoPlayerController: NSObject {
     private let taskExecutor: TasksExecutorProtocol.Type
     private let multicastDelegate = ApiVideoPlayerControllerMulticastDelegate()
     private var playerItemFactory: ApiVideoPlayerItemFactory?
+    private var storedVideoSpeed: Double?
```

Comment:
I guess you have to keep this value because calling play reset rate to `1.0`. Could you explain in a comment? Have you tried to use `defaultRate` to avoid keeping this value: https://developer.apple.com/documentation/avfoundation/avplayer/3929373-defaultrate?

If there is no other choice than storing this value, could you make it non optional? Like initialize it with AVPlayer default rate?